### ISSUE TITLE
"Please enter a valid path to a Plink executable." When selecting "External Open SS

### ISSUE THREAD

**Issue by del-leehopper** [git-for-windows/git#4831]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options

I have tried:
2.43.0-64-bit.exe
2.44.0-rc2-64-bit.exe

git version 2.44.0.rc2.windows.1
cpu: x86_64
built from commit: 0db3af6cead6855bb0d72b7149f5c2d9adb5f835
sizeof-long: 4
sizeof-size_t: 8
shell-path: /bin/sh
feature: fsmonitor--daemon
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

Microsoft Windows [Version 6.3.9600] (Server 2012r2)
Microsoft Windows [Version 10.0.19045.4046]

```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
> type "$env:USERPROFILE\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

Editor Option: Notepad++
Custom Editor Path:
Default Branch Option: master
Path Option: Cmd
SSH Option: ExternalOpenSSH
Tortoise Option: false
CURL Option: WinSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Git Pull Behavior Option: Merge
Use Credential Manager: Enabled
Performance Tweaks FSCache: Enabled
Enable Symlinks: Disabled
Enable Pseudo Console Support: Disabled
Enable FSMonitor: Disabled
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

On the Windows Server machine, OpenSSH is not installed.
On the Windows 10 Machine, Open SSH is installed.

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

CMD

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
Error occurred during installation. I chose the "Use external OpenSSH" option and got a prompt when clicking "Next".

```
 - What did you expect to occur after running these commands?

A prompt stating that OpenSSH wasn't installed

 - What actually happened instead?

A prompt that states: 
"Please enter a valid path to a Plink executable."
![Git - OpenSSH Error](https://github.com/git-for-windows/git/assets/78899539/f0ee99be-6b5f-47ac-b2b5-6fc702de7edf)

However, when installing on Windows 10 that does have OpenSSH installed, it installed perfectly fine.

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

N/A

**Comment by dscho** [comment]
The code that needs to be fixed is [here](https://github.com/git-for-windows/build-extra/blob/e85444a225764d80b35c3f817062d014d310b41b/installer/install.iss#L2564-L2569).

Basically, there needs to be another `if RdbSSH[GS_ExternalOpenSSH].Checked ... else ...` conditional, having two different messages instead of [the hardcoded Plink error](https://github.com/git-for-windows/build-extra/blob/e85444a225764d80b35c3f817062d014d310b41b/installer/install.iss#L2568).

**Comment by del-leehopper** [comment]
Yes I saw that too. Wasn't too familiar with how this software worked so I didn't want to make a fool of myself. 

Should be quite a simple fix I think and will hopefully stop others having a confusing experience too (I thought OpenSSH was installed on this server, but apparently not - a relevant prompt would have saved time and confusion).

**Comment by dscho** [comment]
> Should be quite a simple fix

I won't have time to implement this (and to test, because I'd have to set up a working Plink first, which I don't have). Do you have the time, @del-leehopper?

**Comment by del-leehopper** [comment]
Unfortunately not, my available time is also very limited at the moment, plus I have never used Plink before. Normally, I would be happy to learn, but I really don't have the capacity at the moment, sorry.
